packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6064,6 +6070,11 @@ export default class Meeting extends StatelessWebexPlugin {
   public roapMessageReceived = (roapMessage: RoapMessage) => {
     const mediaServer = MeetingsUtil.getMediaServer(roapMessage.sdp);
 
+    if (this.isMultistream && mediaServer !== 'homer') {
```

Comment:
can we make a MEDIA_SERVER constant somewhere? I'm not sure if we check for homer in the SDP for example anywhere else?

Reply:
We don't check anywhere else, this is the only place for homer. There is a MEDIA_SERVER constant in SDK tests that has 3 values, LINUS, HOMER, CALLIOPE - which is not really correct because Linus and Homer are part of Calliope and also there is an issue that getMediaServer() only works correctly for Homer, making it distinguish between Linus and Edonus requires more effort, not even sure if we are able to tell them apart based on the SDP, so this is not something I want to be doing now in this PR.